sig-scalability/block_merges.md

```diff
@@ -0,0 +1,51 @@
+# Blocking PR merges in the event of regression.
+
+As mentioned in the charter, SIG scalability has a right to block all PRs
+from merging into the relevant repos. This document describes the underlying
+"Rules of engagement" of this process and the rationale why this is needed.
+
+### Rules of engagement.
+The rules of engagement for blocking merges are as following:
+
+- Observe as scalability regression on one of release-blocking test suites.
```

Comment:
Which suites specifically?

How is a regression observed?  The suites have been continually failing, so it doesn't seem like it's a green-to-red transition.